clang/test/Driver/sycl-offload-amdgcn.cpp

```diff
@@ -9,6 +9,11 @@
 // RUN: | FileCheck -check-prefix=CHK-ARCH %s
 // CHK-ARCH: error: missing AMDGPU architecture for SYCL offloading; specify it with '-Xsycl-target-backend --offload-arch'
 
+// RUN: not %clangxx -### -std=c++11 -target x86_64-unknown-linux-gnu -fsycl \
+// RUN: -fsycl-targets=spir64,amdgcn-amd-amdhsa %s 2>&1 \
```

Comment:
Is there a similar error for `-fsycl-targets=spir64,nvptx-nvidia-cuda` ?

Reply:
No, this is an AMD-specific error. I'm not 100% but it seems as though we always compile for a generic "nvidia/cuda" target, and I presume the PTX compiler does the arch-specific parts.